### PR TITLE
Proposal about KubeEdge support kubectl logs and exec command

### PR DESCRIPTION
**Pull request by kadisi**
Signed-off-by: zhangjie <iamkadisi@163.com>

**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind test
> /kind failing-test
> /kind feature

/kind feature

**What this PR does / why we need it**:
Ref: https://github.com/kubeedge/kubeedge/pull/1014

**Which issue(s) this PR fixes**:

Fixes: 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by kevin-wangzefeng** [comment]
Ref: #1477

**Comment by humbleengineer** [comment]
@kadisi For our intended use we'll be very excited if/when `kubectl logs` is supported.  However for security reasons we'd like to have the option to disable `kubectl exec`.  Will there be options in `cloudcore.yaml` to explicitly enable streaming channels `logs`, `metrics`, and `exec`?  Apologies I'm pretty new to K8s, so if there's a better way to disable `exec` please let me know.

**Comment by daixiang0** [comment]
lgtm

File: docs/proposals/exec-logs.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+```
+
+The flowchart is as follows:
+<img src="../images/proposals/kubeedge-exec-con4.png">
+
+
+## Detail Design
+
+working in process ...
```
Comment (chendave) [inline docs/proposals/exec-logs.md:141]: @daixiang0 can you update this as well?

**Comment by chendave** [comment]
also echo to @humbleengineer 's comments, is that way to opt-in/out this feature?

File: docs/proposals/exec-logs.md (hunk 1 above)
Comment (daixiang0) [reply docs/proposals/exec-logs.md:141]: I do not have premission to wirte this pr :(

**Comment by daixiang0** [comment]
@chendave it is disabled by default.

**Pull request closed without merging**